### DISCUSSION TITLE
DAI 2024-2025 - My Docker Compose application - Basile Buxtorf

### DISCUSSION THREAD

**Discussion by BasileBux** [heig-vd-dai-course/heig-vd-dai-course#405] [Show and tell]
https://github.com/BasileBux/heig-vd-dai-course-java-ios

![07-10-24_10h37m57s](https://github.com/user-attachments/assets/8cfed0cc-2078-4c2d-835c-d06076e9c8c2)

**Comment by ludelafo** [comment]
Thank you!

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
